sql/expression/function/srid.go

```diff
@@ -33,11 +33,6 @@ var _ sql.FunctionExpression = (*SRID)(nil)
 
 var ErrInvalidSRID = errors.NewKind("There's no spatial reference with SRID %d")
 
-const (
-	CartesianSRID  = uint32(0)
-	GeoSpatialSRID = uint32(4326)
```

Comment:
I moved this to sql layer since it's used in all layers of sql not only functions